test/0.1.1-rc.12/domain.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('Domain Layer', () => {
   let accounts;
   const orgName = getOrgName();
   const tokenName = orgName + ' Token';
-  const tokenSymbol = orgName[0] + orgName.split(' ')[1][0] + 'T';
+  const tokenSymbol = 'MY';
```

Comment:
why this change?

Reply:
Because Jordan changed the DAO name and the token name and symbol to be constant